cc1120_arduino_tests/cc1120_spi.h

```diff
@@ -0,0 +1,203 @@
+#ifndef SPI_CC1120_H
+#define SPI_CC1120_H
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#define R_BIT 1 << 7
+#define BURST_BIT 1 << 6
+
+struct cc_status {
+  uint8_t res : 4;
+  uint8_t state : 3;
```

Comment:
It might help to have an enum with all the 8 possible states (TX, RX, Fast TX, etc.)